<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SYSU_PRCameraClass</class>
 <widget class="QMainWindow" name="SYSU_PRCameraClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>836</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1092</width>
    <height>836</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SYSU Semantic Video Recognition </string>
  </property>
  <property name="windowIcon">
   <iconset resource="sysu_prcamera.qrc">
    <normaloff>:/SYSU_PRCamera/sysu1.ico</normaloff>:/SYSU_PRCamera/sysu1.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="cameralbl">
       <property name="minimumSize">
        <size>
         <width>681</width>
         <height>471</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>271</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="utcLbl_3">
            <property name="minimumSize">
             <size>
              <width>51</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cameraId"/>
          </item>
          <item>
           <widget class="QPushButton" name="lunchCamera">
            <property name="minimumSize">
             <size>
              <width>61</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>61</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Launch</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="timeLbl">
          <property name="text">
           <string>Current Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="utcLbl">
          <property name="text">
           <string>UTC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="utcLbl_2">
          <property name="text">
           <string>Cur. Photo Class</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="classCombo">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>252</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>231</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>231</width>
              <height>71</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="sysu_prcamera.qrc">:/Images/Resources/147832221417159333.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="minimumSize">
       <size>
        <width>1051</width>
        <height>201</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Console</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextEdit" name="output">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Features</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTextEdit" name="featureList">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 85, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="autoFormatting">
           <set>QTextEdit::AutoNone</set>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::WidgetWidth</enum>
          </property>
          <property name="overwriteMode">
           <bool>true</bool>
          </property>
          <property name="cursorWidth">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="actionRunCamera"/>
    <addaction name="actionCloseCamera"/>
    <addaction name="separator"/>
    <addaction name="actionClearSettings"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSavePhoto"/>
    <addaction name="separator"/>
    <addaction name="actionExportSamples"/>
   </widget>
   <widget class="QMenu" name="menuAlgo">
    <property name="title">
     <string>ML</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionTrainSVM"/>
    <addaction name="actionLoadSVM"/>
    <addaction name="separator"/>
    <addaction name="actionSVMPredict"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menuFile"/>
   <addaction name="menuAlgo"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRunCamera"/>
   <addaction name="actionCloseCamera"/>
   <addaction name="separator"/>
   <addaction name="actionSavePhoto"/>
   <addaction name="actionExportSamples"/>
   <addaction name="separator"/>
   <addaction name="actionTrainSVM"/>
   <addaction name="actionLoadSVM"/>
   <addaction name="separator"/>
   <addaction name="actionSVMPredict"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRunCamera">
   <property name="text">
    <string>Run Camera</string>
   </property>
  </action>
  <action name="actionCloseCamera">
   <property name="text">
    <string>Close Camera</string>
   </property>
  </action>
  <action name="actionSavePhoto">
   <property name="text">
    <string>Take Sample (Ctrl+T)</string>
   </property>
  </action>
  <action name="actionClearSettings">
   <property name="text">
    <string>Clear Settings</string>
   </property>
  </action>
  <action name="actionExportSamples">
   <property name="text">
    <string>Export Samples</string>
   </property>
  </action>
  <action name="actionTrainSVM">
   <property name="text">
    <string>Train SVM</string>
   </property>
  </action>
  <action name="actionLoadSVM">
   <property name="text">
    <string>Load SVM</string>
   </property>
  </action>
  <action name="actionSVMPredict">
   <property name="text">
    <string>SVM Predict (Ctrl+P)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sysu_prcamera.qrc"/>
 </resources>
 <connections/>
</ui>
